src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -186,6 +198,12 @@
             :show-keyboard?  false
             :value           (controlled-input/input-value input-state)
             :on-swap         swap-between-fiat-and-crypto}]
+          (when lock-higher-than-send-amount?
```

Comment:
what is lock-higher than. should be like `locked-greater-then-send-amount?` ?

Reply:
thanks!